src/components/Map.js

```diff
@@ -0,0 +1,62 @@
+import GoogleMap from 'google-map-react'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import Cluster from './Cluster'
+import Location from './Location'
+
+const Map = (props) => {
+  const {
+    bootstrapURLKeys,
+    view,
+    markerData,
+    showLocations,
+    onZoomAnimationEnd,
+    onDragEnd,
+    onClusterClick,
+    onLocationClick,
+  } = props
+
+  return (
+    <GoogleMap
+      bootstrapURLKeys={bootstrapURLKeys}
+      center={view.center}
+      zoom={view.zoom}
+      onZoomAnimationEnd={onZoomAnimationEnd}
+      onDragEnd={onDragEnd}
+    >
```

Comment:
(See below with the props. Consider just rendering both and leaving it up to the user of the component to clear out the opposite list.)